### ISSUE TITLE
Create a SVM example with scikit-learn

### ISSUE THREAD

**Issue by Rishit-dagli** [Rishit-dagli/Python3-by-practice#76]
In #68 the [Breast Cancer Wisconsin (Diagnostic) Dataset](https://github.com/Rishit-dagli/Python3-by-practice/tree/master/Machine%20Learning/sklearn%20examples/Support%20Vector%20Machines(SVM)/Breast%20Cancer%20Wisconsin%20(Diagnostic)%20Dataset) as a sample Logistic Regression dataset was added by @Avinashshah099 . I am looking for contributors who could add a notebook with an aim to show how Logistic Regression works with `sklearn` to predict `diagnosis`. At the start of the PR description include `Closes #76` for automatic issue linking.

The notebook is expected to be placed under `Machine Learning/sklearn examples/Support Vector Machines(SVM)/SVM.ipynb`

### Include Author name

You would be expected to include under the heading:

```
_Originally Contributed by [GitHub-Username](https://github.com/GitHub-Username)_
```

### Include buttons for accessibility
Include a `Run in Google Colab` and `View Source on GitHub` buttons

#### Example (taken from the [Linear Regression Notebook](https://github.com/Rishit-dagli/Python3-by-practice/blob/master/Machine%20Learning/sklearn%20examples/Linear%20Regression/Linear%20Regression%20.ipynb) in this repo)-

_Notebook orignially contributed by: [Aditya-1500](https://github.com/Aditya-1500)_

<table class="tfo-notebook-buttons" align="left">
  <td>
    <a target="_blank" href="https://colab.research.google.com/github.com/Rishit-dagli/Python3-by-practice/blob/master/Machine%20Learning/sklearn%20examples/Linear%20Regression/Linear%20Regression%20.ipynb"><img src="https://www.tensorflow.org/images/colab_logo_32px.png" />Run in Google Colab</a>
  </td>
  <td>
    <a target="_blank" href="https://github.com/Rishit-dagli/Python3-by-practice/blob/master/Machine%20Learning/sklearn%20examples/Linear%20Regression/Linear%20Regression%20.ipynb"><img src="https://www.tensorflow.org/images/GitHub-Mark-32px.png" />View source on GitHub</a>
  </td>
</table>

<br>
<br>
<br>

#### Code to generate buttons

```
<table class="tfo-notebook-buttons" align="left">
  <td>
    <a target="_blank" href="https://colab.research.google.com/github/Rishit-dagli/Python3-by-practice/blob/master/Machine%20Learning/sklearn%20examples/Support%20Vector%20Machines(SVM)/SVM.ipynb"><img src="https://www.tensorflow.org/images/colab_logo_32px.png" />Run in Google Colab</a>
  </td>
  <td>
    <a target="_blank" href="https://github.com/Rishit-dagli/Python3-by-practice/blob/master/Machine%20Learning/sklearn%20examples/Support%20Vector%20Machines(SVM)/SVM.ipynb"><img src="https://www.tensorflow.org/images/GitHub-Mark-32px.png" />View source on GitHub</a>
  </td>
</table>
```
### Include a short intro

Below the above-mentioned notebook buttons, be sure to add a small intro showing what the notebook does.

### Include clear Dataset Credits

In the notebook, intro be sure to add clear credits to the dataset owner.

**Comment by Rishit19** [comment]
@Rishit-dagli please assign me this issue.

**Comment by Rishit-dagli** [comment]
Solved by #83
